invenio_admin/ext.py

```diff
@@ -166,6 +169,42 @@ def lazy_base_template():
         return state
 
     @staticmethod
+    def _override_csp_header(app):
+        """Flask-Admin extension is not CSP compliant (see: https://github.com/flask-admin/flask-admin/issues/1135)
+        To avoid UI malfunctions, the CSP header (globally set on each request by Talisman extension) must be
+        overridden and removed.
+        Remove this code if and when Flask-Admin will be completely CSP compliant
+        """
+
+        @app.before_request
+        def _disable_talisman_csp_admin_views():
+            """Override Talisman configuration for admin views by setting the CSP to None before the request.
+            There is no certitude that this @before_request will be executed before Talisman @before_request: this
+            case is covered and fixed by the @after_request.
+            """
+            view_function = current_app.view_functions.get(request.endpoint)
+            if _is_protected_admin_view(view_function):
+                invenio_app = app.extensions['invenio-app']
+                setattr(invenio_app.talisman.local_options, 'content_security_policy', None)
```

Comment:
Shouldn't this be done in `after_request`? At this point it's not clear if talisman's `before_request` has been applied or not. On the other hand in `after_request` it's certain that the `talisman.local_options` object exists.

Another thing is that this patch assumes that `invenio-app` is being used (which might not be always the case), so probably an `if` clause might be necessary. @lnielsen, is there a plan to use `invenio-app` by default with the `app` fixture in `conftest.py` of all Invenio modules?